components/match2/commons/match_summary_ffa.lua

```diff
@@ -44,7 +44,9 @@ local MATCH_OVERVIEW_COLUMNS = {
 	{
 		class = 'cell--status',
 		show = function(match)
-			return Table.isNotEmpty(match.extradata.status)
+			return Table.any(match.extradata.status, function(_, value)
```

Comment:
```suggestion
			return Table.any(match.extradata.status or {}, function(_, value)
```
there will be wikis that do not use this, so making sure to not error on nil is imo important here